packet/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -64,9 +64,8 @@ resource "null_resource" "copy-controller-secrets" {
   }
 }
 
-# Secure copy bootkube assets to ONE controller and start bootkube to perform
-# one-time self-hosted cluster bootstrapping.
-resource "null_resource" "bootkube-start" {
+# Secure copy bootkube assets to ONE controller
+resource "null_resource" "copy-assets-dir" {
```

Comment:
In commit message:
>Due to the move to helm chart , there does not exist any directory named
  manifests or manifess-networking.

Typo + I'm don't fully understand this sentence. Could you elaborate @ipochi ?

Reply:
> In commit message:
> 
> > Due to the move to helm chart , there does not exist any directory named
> > manifests or manifess-networking.
> 
> Typo + I'm don't fully understand this sentence. Could you elaborate @ipochi ?

reworded the commit message to reflect the reasons behind the change.
Please let me know if the reasons make sense now.

Reply:
Yes, it makes more sense now. Thanks!